CONTRIBUTING.md

```diff
@@ -196,13 +228,6 @@ which appears in the documentation, this ensures us that our examples
 should always compile, and our documentation has a better chance of
 staying up-to-date.
 
-### Publishing the site to github.
-
-The `git.remoteRepo` variable in `docs/build.sbt` controls which
-repository you will push to. Ensure that this variable points to a
-repo you wish to push to, and that it is one for which you have push
-access, then run `sbt ghpagesPushSite`
-
 ### Write examples
```

Comment:
I guess we can write here: 
One of the best ways to provide examples is doctest, here is an [example of doctest examples in cats](https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/Functor.scala#L19-L33).